kornia/geometry/boxes.py

```diff
@@ -55,6 +75,61 @@ def _boxes_to_polygons(
     return polygons
 
 
+def _boxes_to_quadrilaterals(
+    boxes: torch.Tensor, mode: str = "xyxy", validate_boxes: bool = True
+) -> torch.Tensor:
+    """Convert from boxes to quadrilaterals."""
+    mode = mode.lower()
+
+    if mode.startswith("vertices"):
+        batched = boxes.ndim == 4
+        if not (3 <= boxes.ndim <= 4 and boxes.shape[-2:] == torch.Size([4, 2])):
+            raise ValueError(f"Boxes shape must be (N, 4, 2) or (B, N, 4, 2) when {mode} mode. Got {boxes.shape}.")
+    elif mode.startswith("xy"):
+        batched = boxes.ndim == 3
+        if not (2 <= boxes.ndim <= 3 and boxes.shape[-1] == 4):
+            raise ValueError(f"Boxes shape must be (N, 4) or (B, N, 4) when {mode} mode. Got {boxes.shape}.")
+    else:
+        raise ValueError(f"Unknown mode {mode}")
+
+    boxes = boxes if boxes.is_floating_point() else boxes.float()
+    boxes = boxes if batched else boxes.unsqueeze(0)
+
+    if mode.startswith("vertices"):
+        if mode == "vertices":
+            # Avoid passing reference
+            quadrilaterals = boxes.clone()
+        elif mode == "vertices_plus":
+            quadrilaterals = boxes.clone()  # TODO: perform +1
+        else:
+            raise ValueError(f"Unknown mode {mode}")
+        validate_boxes or validate_bbox(quadrilaterals)
```

Comment:
i guess here an assignment is missing

Reply:
https://deepsource.io/gh/kornia/kornia/run/3bef9ab6-4222-4d5d-9ff9-e1e985cf66f5/python/PYL-W0106

Reply:
Nope. Just to validate if the box not existed.